docs/authentication-backends.md

```diff
@@ -50,23 +46,23 @@ To enable it:
    docs](https://www.netlify.com/docs/authentication-providers/#using-an-authentication-provider).
 2. Add the following lines to your `config.yml` file:
 
-   ``` yaml
-   backend:
-     name: github
-     repo: owner-name/repo-name # Path to your Github repository
-   ```
+    ``` yaml
+    backend:
+      name: github
+      repo: owner-name/repo-name # Path to your Github repository
+    ```
```

Comment:
I had to indent the code blocks one more tab to make them properly display in the numbered list, here and in the Git Gateway steps.
(Side note: we should take a look at how the rich text editor marks up code blocks inside of lists, and be sure they produce the desired output in most cases.)